zenrock/chain.json

```diff
@@ -0,0 +1,114 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "zenrock",
+    "status": "live",
+    "website": "https://zenrocklabs.io",
+    "network_type": "mainnet",
+    "chain_type": "cosmos",
+    "pretty_name": "Zenrock Mainnet",
+    "chain_id": "diamond-1",
+    "bech32_prefix": "zen",
```

Comment:
`zen` is taken by Zen protocol?? https://github.com/satoshilabs/slips/blob/master/slip-0173.md
At line: https://github.com/satoshilabs/slips/blob/master/slip-0173.md?plain=1#L293

Reply:
Bech32 prefixes are not meant to be exclusive, no? 
We haven't seen it in the chain registry here either

Reply:
This topic was recently discussed and there did turn out to be some issues when two chains shared a bech32 prefix, although that may have been limited to bech32 prefixes actually on the Chain Registry.
However we generally require that all chains register their prefix to that SLIP0173 page once it reaches mainnet, and its probably best to not have any collisions.

Reply:
I understand your concern of having a collusion-free repo but some heads-up would be great. There is no information on this requirement and the dependency of the SLIP0173 page here on the repo. We can't change our prefix anymore as we are already on mainnet. 

Further, I saw that there are other projects with a shared prefix on this repo (like terra) and the other chain in question is not listed here and not an IBC-enabled chain like us.  

Can you further explain what ind of issues where occurring with two chains sharing a bech32 prefix?

Reply:
I understand the frustration, and are looking to reach consensus on this ASAP.

I don't recall too well what the issue was, but some apps were using the bech32 prefix as a reliable index and would load metadata from the wrong chain (e.g., when users try to view Cosmos Hub data, the app would show data relating to some other chain). Apparently there were multiple apps with issues like this, but I'm not certain.